resources/uk/gov/hmcts/gradle/.dockerignore_test

```diff
@@ -0,0 +1,8 @@
+*
+
+!gradlew
+!build.gradle
+!runTests.sh
+!.gradle
+!gradle
+!src
```

Comment:
this isn't going to work for a lot of projects that have multi module builds setup

Reply:
> this isn't going to work for a lot of projects that have multi module builds setup

What would be a good test project?